stdlib/test/builtin/test_int.mojo

```diff
@@ -195,6 +195,54 @@ def test_int_uint():
     assert_equal(0, int(u2))
 
 
+def test_lt():
```

Comment:
Instead of separate functions, could you please just put these into a single `test_comparisons`? We don't have a consistent strategy for testing these, so for now I would like to reduce the number of test functions.